home-assignments/session2/cli.py

```diff
@@ -0,0 +1,108 @@
+""""
+Ze'ev 10/11/2018
+Exercises for the OOPS School 3, Coding in Python, Session 2, Assignment 1
+
+Command line arguments follow the following format -
+    `python cli.py [] CITY`
+CITY is a mandatory argument
+Options:
+    [--period <num-of-days> -f]
+num-of-days may be in format TODAY+n
+num-of-days default value is TODAY == 1
+Use -f to override default to Fahrenheit measurements
+"""
+
+import click
+import requests
+from weather import Weather, Unit
+
+unit_dict = {
+    False: (Unit.CELSIUS, 'Celsius'),
+    True:  (Unit.FAHRENHEIT, 'Fahrenheit')
+}
+
+
+def parse_unit(unit_flag):
```

Comment:
This function looks like it lacks any logic. Try to think of the real name of this function and then maybe you will understand what it's actual role is

Reply:
OK